src/pybind/mgr/dashboard/controllers/orchestrator.py

```diff
@@ -94,3 +96,15 @@ class OrchestratorService(RESTController):
     def list(self, hostname=None):
         orch = OrchClient.instance()
         return [service.to_json() for service in orch.services.list(None, None, hostname)]
+
+
+@ApiController('/orchestrator/osd', Scope.OSD)
+class OrchestratorOsd(RESTController):
+
+    @raise_if_no_orchestrator
+    def create(self, drive_group, all_hosts=None):
```

Comment:
Currently SSH orchestrator throws a `RuntimeError` exception which is not handled correctly which results in an ugly and unusable error message in the UI. Using `handle_orchestrator_error` will fix this.
```suggestion
    @handle_orchestrator_error('osd')
    def create(self, drive_group, all_hosts=None):
```

Reply:
That's a bug in the orchestrator and not in the UI. The contract is: orchestrator.OrchestratorError errors are user facing errors. All other errors are bugs in the orchestrator.